api/controllers/services.js

```diff
@@ -107,27 +111,43 @@ exports.service_edit = (req, res, next) => {
 
 exports.service_delete = (req, res, next) => {
     var serviceId = mongoose.Types.ObjectId(req.body.id);
```

Comment:
Curious about this line. I think I've deleted it in the past, but I'm not sure. I believe if it throws an exception if the user errantly passes a string that's not a perfectly formatted mongoose object ID. It may be preferable to have the ID go in directly, and let the error handler take care of the potential issues.

Reply:
theoretically the user wouldn't be passing the id as it would be the fronted passing the id of the service that the user wishes to edit(so that shouldn't be to much of an issue as it should be done automatically), also if it is unneeded can get rid of/edit it after all the service routes are working (could have been one of the changes you did to the cleanup branch that i closed the request on)

Reply:
You're right, first things first. Just something to keep in mind.